### PR TITLE
Copy Croatian tax rates in EU localization packs

### LINKED ISSUE #16336: Copy Croatian tax rates in EU localization packs
**Is your feature request related to a problem? Please describe:**

So far, Croatia was not considered part of the EU in PrestaShop. It has been fixed for version 1.7.7. (cf. #15618) so Croatia and its tax rates must be reported in all European packs.
 
**Describe the solution you'd like:**

🇭🇷 _What are the Croatian tax rates again?_

25% Standard Rate
13% Reduced Rate
5% Reduced Rate
0% Zero Rate

🇪🇺 _What European packs are concerned?_ 

Austria / Italy / Belgium / Latvia / Bulgaria / Lithuania
Croatia / Luxembourg / Cyprus / Malta / Czechia / Netherlands
Denmark / Poland / Estonia / Portugal / Finland / Romania
France / Slovakia / Germany / Slovenia / Greece / Spain
Hungary / Sweden / Ireland / United Kingdom

**Specifications:**

As a merchant, I want to be able to sell all over Europe without having to import all European taxes manually. 

Following this statement, a German merchant must be able to sell his/her products to Croatia at the maximum tax rate of 25%. So his/her store must, by default, contains:

- all German tax rates (here, 19% and 7%)
- the Croatian tax rate (here, 25%)

Each European pack must contain the other European tax rates.

### PR DESCRIPTION
**Pull request by ziegenberg**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Copy Croatian tax rates in all EU localization packs
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #16336
| How to test?  |

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16349)

### CODE REVIEW

File: localization/mt.xml [hunk 1]
```diff
@@ -104,92 +106,94 @@
...
-  <units>
-    <unit type="weight" value="kg"/>
-    <unit type="volume" value="L"/>
-    <unit type="short_distance" value="cm"/>
-    <unit type="base_distance" value="m"/>
-    <unit type="long_distance" value="km"/>
-  </units>
-</localizationPack>
+      <taxRule iso_code_country="hr" id_tax="3">
+        <taxRule iso_code_country="es" id_tax="3"/>
```
Comment (Progi1984) [inline localization/mt.xml]: @ziegenberg It seems there are some additional spaces to remove.

**Comment by Progi1984** [comment]
@ziegenberg Thank you for your contribution. A small fix with spaces on mt.xml. Could you give a scenario test for checking behavior before and after this PR ? Thanks

File: localization/mt.xml (hunk 1 above)
Comment (ziegenberg) [reply localization/mt.xml]: A "/" in line 109 was missing. I'll correct my PR.

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
@ziegenberg Thanks for the fix. Finally, Could you give a scenario test for checking behavior before and after this PR ? After that, I will send your PR to the QA Team. Thanks

**Comment by ziegenberg** [comment]
I'll come up with some testing scenario, but it will take me some days. I'm out of office for the next week.

**Comment by Progi1984** [comment]
@ziegenberg Hello, have you got some news about the test scenario ?

**Comment by Robin-Fischer-PS** [comment]
Hi @ziegenberg ! Could you resolve the conflict on ro.xml file ? Thanks !

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @ziegenberg

**Pull request merged**